src/SWP/Bundle/WebRendererBundle/Controller/DefaultController.php

```diff
@@ -25,12 +26,12 @@ class DefaultController extends Controller
      */
     public function indexAction()
     {
-        $pathBuilder = $this->get('swp_multi_tenancy.path_builder');
-        $site = $this->get('swp.repository.site')->find($pathBuilder->build('/'));
-        $homepage = $site->getHomepage();
+        /** @var HomepageBasedTenantInterface $currentTenant */
+        $currentTenant = $this->get('swp_multi_tenancy.tenant_context')->getTenant();
+        $homepage = $currentTenant->getHomepage();
 
-        if (null === $homepage) {
-            throw $this->createNotFoundException('No homepage configured!');
+        if (null !== $homepage) {
```

Comment:
So it should be possible for a tenant not to have a homepage? What is the behaviour?